gossip/src/contact_info.rs

```diff
@@ -18,16 +18,8 @@ use {
     thiserror::Error,
 };
 
-// inline consts to avoid solana-sdk dep
```

Comment:
It will be a minor annoyance if the values below change such that we have to update a couple tests. RPC already depends on gossip so could hypothetically make these `pub`, but I'm guessing you're not looking to introduce that couple

Reply:
these values were pulled out of some phone guy's ass 6-7yrs ago and haven't changed since. i'll take responsibility for any breakage